vue-frontend/src/components/contact/ContactFormFooter.vue

```diff
@@ -107,28 +107,41 @@
-                :class="
-                  invest == `I'll invest`
-                    ? '!tw-text-white/[.6]'
-                    : '!tw-text-white'
-                "
+                class="tw-flex tw-items-center tw-justify-between tw-w-full tw-pb-0 tw-px-0 tw-my-[2rem] tw-mx-0 tw-border-b tw-border-white/[.6] tw-bg-none tw-font-medium tw-text-white/[.6] tw-text-[1rem] tw-leading-[1.1875rem] md:tw-text-[1.375rem] md:tw-leading-[1.6875rem] lg:tw-text-[1.75rem] lg:tw-leading-[2.125rem] tw-whitespace-nowrap tw-transition tw-duration-150 tw-ease-in-out focus:tw-outline-0 active:tw-outline-0 focus:tw-shadow-none active:tw-shadow-none focus:tw-ring-0 active:tw-ring-0 focus:tw-bg-transparent active:tw-bg-transparent active:tw-text-white"
                 type="button"
                 id="invest"
                 name="invest"
                 required
                 :disabled="disableInput"
-                @click="toggleList"
+                @click="
+                  toggleList();
+                  makefloatActive();
+                "
+                @blur="
+                  makeFloatInactive();
+                  deselectOption();
+                "
+                :class="!investSelected ? 'tw-text-white/[.6]' : ''"
               >
                 {{ invest }}
+                <label
+                  class="tw-pb-2 2xl:tw-pb-8"
+                  :class="
+                    floatable
+                      ? 'tw-absolute tw-mt-[5rem] tw-pb-2 tw-mb-[7rem] tw-left-0 tw-z-[2] tw-text-white tw-text-[1rem] tw-leading-[1.1875rem] md:tw-text-[1.375rem] md:tw-leading-[1.6875rem] lg:tw-text-[1.75rem] lg:tw-leading-[2.125rem] tw-transform tw--translate-y-4 tw-origin-[0] tw-scale-75 tw-duration-300 peer-focus:tw-text-white peer-placeholder-shown:tw-scale-100 peer-placeholder-shown:tw-translate-y-0 peer-focus:tw-scale-75 peer-focus:tw--translate-y-4'
+                      : '' || investSelected
+                      ? 'tw-absolute tw-bottom-10 tw-left-0 tw-z-[2] tw-text-white tw-text-[1rem] tw-leading-[1.1875rem] md:tw-text-[1.375rem] md:tw-leading-[1.6875rem] lg:tw-text-[1.75rem] lg:tw-leading-[2.125rem] tw-transform tw--translate-y-4 tw-origin-[0] tw-scale-75 tw-duration-300 peer-focus:tw-text-white peer-placeholder-shown:tw-scale-100 peer-placeholder-shown:tw-translate-y-0 peer-focus:tw-scale-75 peer-focus:tw--translate-y-4'
+                      : ''
+                  "
+                  >I'll invest</label
```

Comment:
![Screenshot from 2023-01-27 17-32-13](https://user-images.githubusercontent.com/69897605/215082014-27a7cdc2-0326-405d-82fe-d18cf0fb597d.png)
Can take ref from https://codepen.io/dannibla/pen/amgRNR
Fix top space. Also top space of `send me NDA ` is too much, make all fields top space identical
Refactor logic accordingly